app/src/main/res/layout/item_skill_group_list.xml

```diff
@@ -2,23 +2,22 @@
 <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:id="@+id/group_parent"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
+    android:layout_height="50dp"
+    android:background="@color/md_white_1000"
     android:orientation="horizontal"
     android:padding="@dimen/padding_small">
 
     <TextView
         android:id="@+id/group"
         android:layout_width="0dp"
-        android:layout_height="wrap_content"
+        android:layout_height="match_parent"
+        android:layout_marginStart="@dimen/margin_small_medium"
         android:layout_weight="1"
         android:gravity="center_vertical"
         android:padding="@dimen/padding_small"
         android:text="@string/sample_group"
-        android:textColor="@color/md_grey_700"
-        android:textSize="@dimen/text_size_moderate"
-        android:textStyle="bold"
-        android:typeface="serif"
-        android:fontFamily="sans-serif"/>
```

Comment:
Why?

Reply:
that's because it's better without it... bold font doesn't look good, same with font family, you can see the difference.